spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/TaskJobService.java

```diff
@@ -126,8 +137,8 @@ JobInstanceExecutions getJobInstance(long id)
 	 * @see org.springframework.batch.admin.service.JobService#stop(Long)
 	 *
 	 * @param jobExecutionId The id of the {@link JobExecution} to stop
-	 * @throws NoSuchJobExecutionException
-	 * @throws JobExecutionNotRunningException
+	 * @throws NoSuchJobExecutionException if no job execution exists for the jobExecutionId.
+	 * @throws JobExecutionNotRunningException if the job execution specified by jobExecutionId can not be restarted.
```

Comment:
Instead of saying `can not be restarted`, we can say the similar way we did above: `if a stop is requested on a job that is not running.` Will fix when merging.